packages/yourloops/components/patient-list/custom-cells.tsx

```diff
@@ -0,0 +1,156 @@
+ */
+
+import React, { type FunctionComponent } from 'react'
+import Box from '@mui/material/Box'
+import ReportProblemIcon from '@mui/icons-material/ReportProblem'
+import { useAuth } from '../../lib/auth'
+import { useTheme } from '@mui/material/styles'
+import EmailIcon from '@mui/icons-material/Email'
+import EmailOpenIcon from '../icons/email-open-icon'
+import Tooltip from '@mui/material/Tooltip'
+import { useTranslation } from 'react-i18next'
+import IconActionButton from '../buttons/icon-action'
+import PersonRemoveIcon from '../icons/person-remove-icon'
+import AccessTimeIcon from '@mui/icons-material/AccessTime'
+import FlagIcon from '@mui/icons-material/Flag'
+import FlagOutlineIcon from '@mui/icons-material/FlagOutlined'
+
+interface FlagCellProps {
+  isFlagged: boolean
+  patientId: string
+}
+
+interface AlarmPercentageCellProps {
+  isAlarmActive: boolean
+  value: number
+}
+
+interface MessageCellProps {
+  hasNewMessages: boolean
+}
+
+interface ActionsCellProps {
+  patientId: string
+  onClickRemove: (patientId: string) => void
+}
+
+export const FlagIconCell: FunctionComponent<FlagCellProps> = ({ isFlagged, patientId }) => {
+  const { flagPatient } = useAuth()
+
+  const onClickFlag = async (): Promise<void> => {
+    await flagPatient(patientId)
+  }
+
+  return (
+    <IconActionButton
+      icon={isFlagged
+        ? <FlagIcon
+          titleAccess="flag-icon-active"
+          aria-label="flag-icon-active"
+        />
+        : <FlagOutlineIcon
+          titleAccess="flag-icon-inactive"
+          aria-label="flag-icon-inactive"
+        />}
+      onClick={onClickFlag}
+    />
+  )
+}
+
+export const PendingIconCell: FunctionComponent = () => {
+  const { t } = useTranslation()
+
+  return (
+    <Tooltip
+      title={t('pending-invitation')}
+      aria-label={t('pending-invitation')}
+    >
+      <Box display="flex">
+        <AccessTimeIcon titleAccess="pending-icon" />
+      </Box>
+    </Tooltip>
+  )
+}
+
+export const AlarmPercentageCell: FunctionComponent<AlarmPercentageCellProps> = ({ isAlarmActive, value }) => {
+  const { user } = useAuth()
+  const theme = useTheme()
+
+  return (
+    <Box
+      display="flex"
+      alignItems="end"
+      sx={{ color: user.isUserHcp() && isAlarmActive ? theme.palette.warning.main : 'inherit' }}
+    >
+      {Math.round(value * 10) / 10}%
+      {user.isUserHcp() && isAlarmActive &&
+        <ReportProblemIcon sx={{ marginLeft: theme.spacing(1) }} color="warning" />
+      }
+    </Box>
+  )
+}
+
+export const MessageCell: FunctionComponent<MessageCellProps> = ({ hasNewMessages }) => {
+  const { t } = useTranslation()
+
+  return (
+    <Tooltip
+      title={t(hasNewMessages ? 'unread-messages' : 'no-new-messages')}
+      aria-label={t(hasNewMessages ? 'unread-messages' : 'no-new-messages')}
+    >
+      <Box display="flex" justifyContent="center">
+        {hasNewMessages
+          ? <EmailIcon titleAccess="unread-messages-icon" color="primary" />
```

Comment:
<!-- Chores are simple tasks that must be done before the subject can be “officially” accepted. Usually, these comments reference some common process. Try to leave a link to the process description so that the reader knows how to resolve the chore.  -->
**💣 chore:** ‏titleAccess should be human-readable ("The patient toto has send you new messages"). If this element has a title access, `EmailOpenIcon` should have one too